lib/Analysis/AffineStructures.cpp

```diff
@@ -683,6 +683,76 @@ LogicalResult FlatAffineConstraints::composeMap(AffineValueMap *vMap) {
   return success();
 }
 
+// Similar to composeMap except that no Value's need be associated with the
+// constraint system nor are they looked at -- since the dimensions and
+// symbols of 'other' are expected to correspond 1:1 to 'this' system. It
+// is thus not convenient to share code with composeMap.
+LogicalResult FlatAffineConstraints::composeMatchingMap(AffineMap other) {
+  assert(other.getNumDims() == getNumDimIds() && "dim mismatch");
+  assert(other.getNumSymbols() == getNumSymbolIds() && "symbol mismatch");
+
+  std::vector<SmallVector<int64_t, 8>> flatExprs;
+  FlatAffineConstraints localCst;
+  if (failed(getFlattenedAffineExprs(other, &flatExprs,
+                                     &localCst))) {
```

Comment:
This fits on one line, can you reflow?

Reply:
Done, thanks. But looks like this was committed.